weg-li/Features/MailView.swift

```diff
@@ -51,7 +51,7 @@ struct MailView: UIViewControllerRepresentable {
     
     func makeUIViewController(context: UIViewControllerRepresentableContext<MailView>) -> MFMailComposeViewController {
         let vc = MFMailComposeViewController()
-        vc.setToRecipients([report.suggestedublicAffairsOffice?.mail ?? ""])
+        vc.setToRecipients([report.district?.mail ?? ""])
```

Comment:
Should we skip districts without an email address? With guards? https://docs.swift.org/swift-book/ReferenceManual/Statements.html#grammar_guard-statement

Reply:
This does type coercion on the optional district. I think you can get around this by having an emptyobject approach instead of nil.